### DISCUSSION TITLE
what target-index number is displayed.

### DISCUSSION THREAD

**Discussion by kooKanno** [hiukim/mind-ar-js#407] [Q&A]
I would like to know how to get what target-index number is displayed.

**Comment by hiukim** [comment]
register an onTargetFound event on the anchor

**Reply by kooKanno** [reply]
Thank you for answering.

I think it's possible using three.js, but can it be used in projects using a-freme?

**Comment by hiukim** [comment]
same. they are on registered on specific target.

https://hiukim.github.io/mind-ar-js-doc/examples/events-handling#targetfound-and-targetlost

**Reply by kooKanno** [reply]
thanks🙏 The method you described solved the problem.

**Comment by ffd8** [comment]
@hiukim – for the usecase of having multiple-targets, does that mean that each `a-entity` (ie, *#example-target1*, *#example-target2*) would need to have their own `targetFound`/`targetLost` event handlers? Is it possible to have a more global check that one has been found and which one? Why is this useful? I'd be running multiple p5.js sketches as the AR layer, and would want to pause all layers/sketches which aren't visible with `noLoop()` and activate the `targetFound` one based on index with `loop()`.

**Reply by BostonLeeK** [reply]
You can just save last tracked marker index in global / store variable and then use it where you need.

**Reply by BostonLeeK** [reply]
Event is global, you can subscribe on it whenever you want
